airlock/models.py

```diff
@@ -925,6 +925,44 @@ def all_files_reviewed_by_reviewer(self, reviewer: User) -> bool:
     def submitted_reviews_count(self):
         return len(self.submitted_reviews)
 
+    def filegroups_missing_comment_by_reviewer(self, reviewer) -> set[str]:
```

Comment:
Just to say that I really appreciated the clarity of this function. It's the sort of logic that's potentially a bit fiddly but the naming of things and the comments make it incredibly easy to read and see that it's doing the right thing.